tests/PFire.CoreTests/UT_PFireServerTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PFire;
+using PFire.Session;
+using System;
+
+
+namespace PFire.Tests
+{
+    [TestClass()]
+    public class UT_PFireServerTests
+    {
+        [TestMethod()]
+        public void PFireServer_InitializationTests()
+        {
+            PFireServer _testServer = new PFireServer(@"C:\Temp");
```

Comment:
I think there's a way to run the sqlite server in memory instead of needing a file. If anything, we need to avoid writing the c drive at all. If we have to write to the file system, use [this](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettemppath?view=net-5.0&tabs=windows) or something similar